app/src/main/java/com/myperishableplanner/v21001/MainActivity.kt

```diff
@@ -26,20 +21,23 @@ class MainActivity : ComponentActivity() {
         setContent {
             MyPerishablePlannerTheme {
                 // A surface container using the 'background' color from the theme
-                Surface(color = MaterialTheme.colors.background , modifier = Modifier.fillMaxWidth()) {
-                    ExpirationFacts("Android")
+                Surface(
+                    color = MaterialTheme.colors.background,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    expirationFacts("Android")
                 }
             }
         }
     }
 }
 
 @Composable
-fun ExpirationFacts (name: String) {
-    var itemName by remember { mutableStateOf(value = "")}
-    var category by remember { mutableStateOf(value = "")}
-    var expirationDate by remember { mutableStateOf(value = "")}
-    var description by remember { mutableStateOf(value = "")}
+fun expirationFacts(name: String) {
+    var itemName by remember { mutableStateOf(value = "") }
+    var category by remember { mutableStateOf(value = "") }
+    var expirationDate by remember { mutableStateOf(value = "") }
+    var description by remember { mutableStateOf(value = "") }
```

Comment:
Spacing added to follow Kotlin style guide per: https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide